workflows/rnaseq/assets/multiqc/multiqc_config.yml

```diff
@@ -168,11 +168,11 @@ custom_data:
       namespace: "Samples failed mapping threshold"
       format: "{:.2f}"
   fail_strand_check:
-    section_name: "WARNING: Fail Strand Check"
-    description: "List of samples that failed the strandedness check between that provided in the samplesheet and calculated by the <a href='http://rseqc.sourceforge.net/#infer-experiment-py'>RSeQC infer_experiment.py</a> tool."
+    section_name: "Strandedness Checks"
+    description: "This table shows library starandedness as provided in the sample sheet, inferred from a read sample by Salmon (where 'auto' was used in the sample sheet), and inferred by the RSeQC <a href='http://rseqc.sourceforge.net/#infer-experiment-py'>RSeQC infer_experiment.py</a> tool from genomic alignments. Match between suplied (or inferred by Salmon) and RSeQC are indicated by &#9989;, a &#10060; status symbol indicates where a mismatch. Libraries with 'undetermined' strandedness are also flagged, since this can indicate QC problems, including possible genomic DNA contamination."
```

Comment:
```suggestion
    description: "This table shows library strandedness as provided in the sample sheet, inferred from a read sample by Salmon (where 'auto' was used in the sample sheet), and inferred by the RSeQC <a href='http://rseqc.sourceforge.net/#infer-experiment-py'>RSeQC infer_experiment.py</a> tool from genomic alignments. Match between suplied (or inferred by Salmon) and RSeQC are indicated by &#9989;, a &#10060; status symbol indicates where a mismatch. Libraries with 'undetermined' strandedness are also flagged, since this can indicate QC problems, including possible genomic DNA contamination."
```

Reply:
Thanks!